assets/styles/_variables.scss

```diff
@@ -26,5 +26,33 @@ $color-background-light: #f2f6fe;
 $color-background-dark: #091a41;
 $color-background-hero: $color-brand;
 
+:root {
+  // editable colors for WP Customizer
+  --color-brand: #{$color-brand};
+  --color-accent: #{$color-accent};
+  --color-accent-dark: #{darken($color-accent, 10%)};
+  --color-contrast: #{$color-contrast};
+  --color-headline: #{$color-headline};
+  --color-border: #{$color-border};
+  --color-background-light: #{$color-background-light};
+  --color-background-dark: #{$color-background-dark};
+  // end of editable colors
+
+  --color-background: #{$color-background}; // TODO: should it be editable color?
```

Comment:
Since its used in a lot of elements right now would be confusing on what changes exactly if we add it to customizer

Reply:
I agree, don't add it to the customizer for now. We can test it later, if some element will need to change white color we can add new variable for that, something like --color-white